compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -801,9 +815,9 @@ class OMR_EXTENSIBLE CodeGenerator
    int32_t getEstimatedSnippetStart() {return _estimatedSnippetStart;}
    int32_t setEstimatedSnippetStart(int32_t s) {return (_estimatedSnippetStart = s);}
 
-   int32_t getAccumulatedInstructionLengthError() {return _accumulatedInstructionLengthError;}
-   int32_t setAccumulatedInstructionLengthError(int32_t e) {return (_accumulatedInstructionLengthError = e);}
-   int32_t addAccumulatedInstructionLengthError(int32_t e) {return (_accumulatedInstructionLengthError += e);}
+   int64_t getAccumulatedInstructionLengthError() {return _accumulatedInstructionLengthError;}
```

Comment:
Why does the accumulated error need to be an `int64_t`?  The buffer length and snippet start offsets are still 32-bit.

Reply:
Done.